bigtop.bom

```diff
@@ -186,7 +186,7 @@ bigtop {
     'sqoop' {
       name    = 'sqoop'
       relNotes = 'Apache Sqoop v1'
-      version { base = '1.4.7'; pkg = base; release = 3 }
+      version { base = '1.4.7'; pkg = base; release = 4 }
```

Comment:
I don't think we have to increment this number, since version 3 is not officially released yet.